core/src/main/scala/com/salesforce/op/stages/impl/selector/ModelSelector.scala

```diff
@@ -233,9 +240,11 @@ final class SelectedModel private[op]
     case m => setDefault(sparkMlStage, Option(m))
   }
 
-  @transient private lazy val recoveredStage: ModelType = getSparkMlStage() match {
-    case Some(m: PredictionModel[_, _]) => SparkModelConverter.toOPUnchecked(m).asInstanceOf[ModelType]
-    case Some(m: ModelType@unchecked) => m
+  @transient private lazy val recoveredStage: ModelType = (getSparkMlStage(), getLocalMlStage()) match {
+    case (Some(m: PredictionModel[_, _]), _) => SparkModelConverter.toOPUnchecked(m).asInstanceOf[ModelType]
+    case (Some(m: ModelType@unchecked), _) => m
+//    case (None, Some(m: MLeapTransformer)) => SparkModelConverter.toOPUnchecked(m).asInstanceOf[ModelType]
```

Comment:
Should this be uncommented ?

Reply:
it is in the next pr when I make it work :-)